archinstall/lib/configuration.py

```diff
@@ -148,7 +148,15 @@ def preview(selection: str) -> Optional[str]:
 		while True:
 			path = input(
 				_(
-					"Enter a directory for the configuration(s) to be saved (tab completion enabled)\nSave directory: "
+					'''\
+Into which DIRECTORY should the configuration(s) be saved?
```

Comment:
this is goind to output every time someone presses <tab> which is quite annoying
![image](https://github.com/user-attachments/assets/fbae2097-e775-41d8-af9d-74449b0ec859)

I would be better to show this output only once on the top